corehq/apps/cloudcare/static/cloudcare/js/util.js

```diff
@@ -210,9 +210,20 @@ hqDefine('cloudcare/js/util',['hqwebapp/js/initial_page_data', 'integration/js/h
         };
     }
 
+    var addDelegatedClickDispatch = function (linkTarget, linkDestination) {
+        document.addEventListener('click', function (event) {
+            if (event.target.target === linkTarget) {
+                linkDestination(event.target);
+                event.preventDefault();
+            }
+        }, true);
```

Comment:
Nitpick: [These docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) make me think the `true` here is equivalent to passing `{capture: true}`. If that's true, could you use the latter, to be more explicit?

Reply:
Oh, the reason I included it this way was the warning in those docs at the bottom that said 

> Note: useCapture has not always been optional. Ideally, you should include it for the widest possible browser compatibility.

The compatibility chart at the bottom of the docs does suggest that `options` isn't supported in IE, so I figured it was best to use the most compatible version if I didn't have a specific reason to use a newer one.

Reply:
Nah, I just didn't read closely enough. Looks good.